cai_causal_graph/time_series_causal_graph.py

```diff
@@ -1137,16 +1169,28 @@ def adjacency_matrices(self) -> Dict[int, numpy.ndarray]:
         return adjacency_matrices
 
     @property
-    def maxlag(self) -> Optional[int]:
```

Comment:
@matteo-causalens, this is a breaking change for the graph API, should we maybe keep the property for backward compatibility and route it to return the new max_time_lag?

Reply:
I can do it but I am not using it anymore so I don't see it as a big issue for the user. But of course I can add it

Reply:
@mmiklitz Dara team is not using it yet, @tadhgokeeffe are you using it?

Reply:
For now I can leave it to replicate the past correct behaviour